server/features/content_rewrite.feature

```diff
@@ -0,0 +1,485 @@
+Feature: Rewrite content
+
+    @auth
```

Comment:
Can we have a scenario to test if the rewrite goes to same desk and incoming stage? Or in one of the existing tests we can verify desk and stage.